README.md

```diff
@@ -2,10 +2,17 @@
 
 A module for Terasology.
 
-## Planed content
+## How to play?
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+First you have to start the game with GooeysQuests activated and in a Perlin world. When the game is created, you will see a character named Gooey that will ask you something. Depending on your answer, Gooey will change the world adding some structures. Now there are two available structures, the dungeon and the dwarf hall, but they have diferents combinations, so every time gooey add the structure, you will see something new. But be careful, maybe you don’t like these changes.
+
+## How to add more structures?
+
+If you want to add a new structure, first you have to create the structures templates that will form the structure. Then you will have to create at least two java class GooeysQuests /src/ main/ java/ org/ terasology/ gooeysQuests/ quests. One will be the component and the other will be the system that allow Gooey to spawn your structure.
```

Comment:
The paths look very weird. Change it to this format:
\`/src/main/java/org/terasology/...\` . That will make it look like this:
`/src/main/java./...`

Reply:
Further, rephrase the sentence so that the rerader knows *what* he's creating, then he knows *where*. What I mean is this:
"Then you will have to create two java classes: a component and a system, typically in the folder `/src/main/java/...`".